### PR TITLE
Removes test with 3.1 and ICU logic

### PR DESCRIPTION
**Pull request by ForestEckhardt**
## Checklist

* [x] I have viewed, signed, and submitted the Contributor License Agreement.
* [ ] I have linked issue(s) that this PR should close using keywords or the Github UI (See [docs](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue))
* [x] I have added an integration test, if necessary.
* [x] I have reviewed the [styleguide](https://github.com/paketo-buildpacks/community/blob/main/STYLEGUIDE.md) for guidance on my code quality.
* [x] I'm happy with the commit history on this PR (I have rebased/squashed as needed).

### CODE REVIEW

File: integration/fdd_aspnet_test.go [hunk 1]
```diff
@@ -1,86 +0,0 @@
-package integration_test
```
Comment (sophiewigmore) [inline integration/fdd_aspnet_test.go:1]: why don't we want these tests or fixtures anymore?

**Comment by sophiewigmore** [comment]
In integration/init_test.go can we remove:
```
}
var builder struct {
	Local struct {
		Stack struct {
			ID string `json:"id"`
		} `json:"stack"`
	} `json:"local_info"`
}
```

File: integration/fdd_aspnet_test.go (hunk 1 above)
Comment (ForestEckhardt) [reply integration/fdd_aspnet_test.go:1]: It was not testing anything unique compared to the integration/framework_dependent_deployment_test.go test. And the test fixture was a 3.1 test fixture. Same for the integration/fde_aspnet_test.go

**Comment by ForestEckhardt** [comment]
@sophiewigmore I am removing the builder struct now.

**Review by sophiewigmore — APPROVED** [review]

**Pull request merged**